### ISSUE TITLE
White line in label of TextInput in dark mode

### ISSUE THREAD

**Issue by mndev222** [callstack/react-native-paper#3983]
### Current behaviour
Error: White line in label of TextInput in dark mode 

### Expected behaviour
In dark mode label text should be visible, not crossed by white line

### How to reproduce?
My theme:
```ts
import { MD3LightTheme } from 'react-native-paper';

export const appTheme = {
  ...MD3LightTheme, // or MD3DarkTheme
  roundness: 2,
  colors: {
    ...MD3LightTheme.colors,
    primary: '#004c80',
    secondary: '#1d4960',
    whiteish: '#FAFAFA',
    blueish: '#E8F1F5',
    error: '#810000',
    info: '#364939',
    grayish: '#324853',
    myOwnColor: '#BADA55',
  },
};

export type AppThemeProps = typeof appTheme;
```

My component:
```ts
import React, { FC, Ref } from 'react';
import { TextInput } from 'react-native-paper';
import { styled } from 'nativewind';
import { NativeSyntheticEvent, TextInputFocusEventData, ReturnKeyTypeOptions } from 'react-native';

const StyledTextInput = styled(TextInput);

type BaseTextInputProps = {
  iconName: string;
  iconColor: string;
  label: string;
  value: string;
  onBlur: (event: NativeSyntheticEvent<TextInputFocusEventData>) => void;
  onChangeText: (value: string) => void;
  onSubmitEditing?: () => void;
  innerRef: Ref<any>;
  returnKeyType?: ReturnKeyTypeOptions | undefined;
};

export const BaseTextInput: FC<BaseTextInputProps> = ({
  label,
  iconColor,
  iconName,
  value,
  onBlur,
  onChangeText,
  onSubmitEditing = () => {},
  innerRef,
  returnKeyType = 'next',
}) => (
  <StyledTextInput
    value={value}
    ref={innerRef}
    label={label}
    outlineColor="#004c80"
    activeOutlineColor="#004c80"
    autoCapitalize="none"
    returnKeyType={returnKeyType}
    mode="outlined"
    selectionColor="#004c80"
    blurOnSubmit={false}
    onBlur={onBlur}
    onChangeText={onChangeText}
    onSubmitEditing={() => {
      onSubmitEditing();
    }}
    left={<TextInput.Icon icon={iconName} color={iconColor} onPress={() => {}} />}
    outlineStyle={{ borderRadius: 8 }}
  />
);

```

### Preview

![1689765137965](https://github.com/callstack/react-native-paper/assets/124168281/fc286bf1-4b8d-4f9a-9c74-63527610ffea)

### Your Environment

| software                      | version
| --------------------- | -------
| ios                                | x
| android                        | 11
| react-native                | 0.71.8
| react-native-paper    | 5.9.0
| node                            | v18.16.1
| npm                             | 9.5.1
| expo sdk                     | 48.0.18

**Comment by lukewalczak** [comment]
Hey @mndev222, I was trying to move your code into the example app, but unfortunately, I didn't observe your issue. Are you able to create an expo snack?

**Comment by tjaniczek** [comment]
Hi @mndev222 , this should be fixed in v5.10.0. Please try upgrading and let us know if it works 👍

**Comment by mndev222** [comment]
> Hi @mndev222 , this should be fixed in v5.10.0. Please try upgrading and let us know if it works 👍

Sorry for the late reply. It hasn't been fixed (got 5.10.4 version).
![processed-ce9f6015-95ff-48ba-88bc-ae9767621658_wSc6htoY](https://github.com/callstack/react-native-paper/assets/124168281/75aa5aa7-26b2-468f-8c39-332d716c1bbf)
